src/anet.c

```diff
@@ -546,18 +546,26 @@ int anetFdToString(int fd, char *ip, size_t ip_len, int *port, int fd_to_str_typ
     } else {
         if (getsockname(fd, (struct sockaddr *)&sa, &salen) == -1) goto error;
     }
-    if (ip_len == 0) goto error;
```

Comment:
i see that was added in 6d53b2f34b16a4fcd0602cb47cb58c31bd28b9e0 but not sure why.
seems like it's dead code these days.